lib/galaxy/webapps/galaxy/api/library_datasets.py

```diff
@@ -401,8 +401,8 @@ def load(self, trans, payload=None, **kwd):
         path = kwd.get('path', None)
         if path is None:
             raise exceptions.RequestParameterMissingException('The required attribute path is missing.')
-        if isinstance(path, int):
-            raise exceptions.RequestParameterInvalidException('The required attribute path is Integer.')
+        if not isinstance(path, str):
+            raise exceptions.RequestParameterInvalidException('The required attribute path is not String.')
```

Comment:
```suggestion
            raise exceptions.RequestParameterInvalidException('The required attribute path is not a string.')
```

Reply:
@OlegZharkov Looks like the article got dropped after the rebase?